### PR TITLE
OSDOCS-10605: updated /var/lib/tuned/ references

### PR DESCRIPTION
**Pull request by sheriff-rh**
Version(s):
4.16+

Issue:
https://issues.redhat.com/browse/OSDOCS-10605

Link to docs preview (VPN required):
[Default profiles set on a cluster](https://file.rdu.redhat.com/antaylor/nto-docs-4.16/scalability_and_performance/using-node-tuning-operator.html#custom-tuning-default-profiles-set_node-tuning-operator)
[Custom tuning examples](https://file.rdu.redhat.com/antaylor/nto-docs-4.16/scalability_and_performance/using-node-tuning-operator.html#custom-tuning-example_node-tuning-operator)

QE review:
- [ ] QE has approved this change.

Additional information:
None

### CODE REVIEW

File: modules/custom-tuning-specification.adoc [hunk 1]
```diff
@@ -348,7 +348,7 @@ ifdef::rosa-hcp-tuning[]
 machine pools.
 endif::rosa-hcp-tuning[]
 
-This functionality takes advantage of `spec.providerID` node object values in the form of `<cloud-provider>://<cloud-provider-specific-id>` and writes the file `/var/lib/tuned/provider` with the value `<cloud-provider>` in NTO operand containers.  The content of this file is then used by TuneD to load `provider-<cloud-provider>` profile if such profile exists.
+This functionality takes advantage of `spec.providerID` node object values in the form of `<cloud-provider>://<cloud-provider-specific-id>` and writes the file `/var/lib/ocp-tuned/provider` with the value `<cloud-provider>` in NTO operand containers.  The content of this file is then used by TuneD to load `provider-<cloud-provider>` profile if such profile exists.
```
[the remark this answers is not in the document]
Comment (sheriff-rh) [reply modules/custom-tuning-specification.adoc]: Fixed in latest push.

**Comment by jmencak** [comment]
Thank you for the changes, Andrew!
/lgtm

File: modules/custom-tuning-specification.adoc [hunk 2]
```diff
@@ -348,7 +348,7 @@ ifdef::rosa-hcp-tuning[]
 machine pools.
 endif::rosa-hcp-tuning[]
 
-This functionality takes advantage of `spec.providerID` node object values in the form of `<cloud-provider>://<cloud-provider-specific-id>` and writes the file `/var/lib/tuned/provider` with the value `<cloud-provider>` in NTO operand containers.  The content of this file is then used by TuneD to load `provider-<cloud-provider>` profile if such profile exists.
+This functionality takes advantage of `spec.providerID` node object values in the form of `<cloud-provider>://<cloud-provider-specific-id>` and writes the file `/var/lib/ocp-tuned/provider` with the value `<cloud-provider>` in NTO operand containers. The content of this file is then used by TuneD to load `provider-<cloud-provider>` profile if such profile exists.
```
Comment (JoeAldinger) [inline modules/custom-tuning-specification.adoc:351]:
```suggestion
This functionality takes advantage of `spec.providerID` node object values in the form of `<cloud_provider>://<cloud-provider-specific-id>` and writes the file `/var/lib/ocp-tuned/provider` with the value `<cloud_provider>` in NTO operand containers. The content of this file is then used by TuneD to load `provider-<cloud_provider>` profile if such profile exists.
```

Comment (JoeAldinger) [reply modules/custom-tuning-specification.adoc:351]: Looking at the example, I think these are user replaced values that require `_`.  I wasn't sure if the <cloud_provider_specific_id> needed to be changed or not as well.

**Review by JoeAldinger — COMMENTED** [review]
Small question about user replaced values

**Comment by sheriff-rh** [comment]
@liqcui  can you review when you have a chance? Thanks.

**Comment by sheriff-rh** [comment]
Merging.

**Pull request merged**